tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py

```diff
@@ -67,6 +67,11 @@ def test_pfc_pause_single_lossy_prio(snappi_api,                # noqa: F811
     snappi_extra_params = SnappiTestParams()
     snappi_extra_params.multi_dut_params.multi_dut_ports = snappi_ports
 
+    flow_factor = 1
+
+    if snappi_extra_params.multi_dut_params.multi_dut_ports[0]['duthost'].facts()['asic_type'] == 'cisco-8800':
```

Comment:
can we also check if port speed is 400G?

Reply:
updated